lib/process-services/task-list/components/start-task.component.html

```diff
@@ -40,11 +40,11 @@
                                 (focusout)="onDateChanged($event.srcElement.value)"
                                 placeholder="{{'ADF_TASK_LIST.START_TASK.FORM.LABEL.DATE'|translate}}"
                                 [(ngModel)]="startTaskmodel.dueDate"
-                                id="date_id">
+                                id="date_id"
+                                (ngModelChange)="onDateChanged($event)">
```

Comment:
It is going to raise 2 events at the same time? there's a `focusout` handler above